src/routes/appointments.js

```diff
@@ -1,12 +1,23 @@
 const router = require('express').Router();
 const Joi = require('joi');
-const { getAppointmentsByDate } = require('../controllers/appointments');
 const wrapper = require('../middleware/wrapper');
 const validator = require('../middleware/validator');
 
+const { getAppointmentsByDate } = require('../controllers/appointments');
+const { createAppointment } = require('../controllers/appointments');
+
 const validatorGetByDate = Joi.object({
 	getByDate: Joi.date().required(),
 });
 router.get('/:getByDate', validator.params(validatorGetByDate), wrapper(getAppointmentsByDate));
 
+const validatorCreateAppointment = Joi.object({
+	date: Joi.date().required(),
+	hour: Joi.string().required(),
```

Comment:
Is not number?

Reply:
hour is not number, check the model, it could be "13:50" or "22.00"

Reply:
Get it